### DISCUSSION TITLE
Removal of Dimensions in graph

### DISCUSSION THREAD

**Discussion by RajeshNakkanaboina** [clientIO/joint#2269] [Q&A] [answered]
### Introduction

Hi Team,

![Screenshot (124)](https://github.com/clientIO/joint/assets/134581112/546e3bb1-0e7b-46d7-a05d-36ec1ec485f1)
I want to get rid of the graph's x and y axes as well as the remaining information that is marked (in the screenshot). Can somebody assist me? 

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment, marked as answer]
Use  `boxContent` option. See `ui.Halo`  [documentation](https://resources.jointjs.com/docs/rappid/v3.7/ui.Halo.html#ui.Halo.configuration) for additional info.
```js
new ui.Halo({ boxContent: false });
```

**Reply by RajeshNakkanaboina** [reply]
Thanks Kumilingus
